db/migrate_cards/data/yinyang.json

```diff
@@ -0,0 +1,168 @@
+        "content": "{\"left\":\"_self\",\"right\":{\"type\":\"year\"},\"dir\":\"asc\",\"limit\":\"1\"}"
+      },
+      {
+        "name": "*unsaved list",
+        "type": "Basic",
+        "content": "<p>You have to {{*account links|sign_in}} or {{*account links|sign_up}} to save your changes.</p>",
+        "codename": "unsaved_list"
+      },
+      {
+        "name": "*empty list",
+        "type": "Basic",
+        "content": "",
+        "codename": "empty_list"
+      },
+      {
+        "name": "*upvotes+*empty list",
+        "type": "HTML",
+        "content": "Drag and drop here to <strong>Vote UP</strong>"
+      },
+      {
+        "name": "*downvotes+*empty list",
+        "type": "Basic",
+        "content": "<p><span>Drag and drop here to </span><strong>Vote DOWN</strong></p>"
+      },
+      {
+        "name": "*upvoted+*right+*structure",
+        "type": "Search",
+        "content": "{\"type\":\"_\",\"linked_to_by\":{\"left\":\"_user\",\"right\":\"*upvotes\"}}"
+      },
+      {
+        "name": "*downvoted+*right+*structure",
+        "type": "Search",
+        "content": "{\"type\":\"_\",\"linked_to_by\":{\"left\":\"_user\",\"right\":\"*downvotes\"}}"
+      },
+      {
+        "name": "*neutral+*right+*structure",
+        "type": "Search",
+        "content": "{\"type\":\"_\",\"not\":{\"linked_to_by\":{\"left\":\"_user\",\"right\":[\"in\",\"*upvotes\",\"*downvvotes\"]}}}"
+      },
+      {
+        "name": "votee search+*right+*structure",
+        "type": "Search",
+        "content": "{\"type\":\"_2\",\"linked_to_by\":{\"left\":\"_user\",\"right\":\"_3\"},\"append\":\"_1\"}"
+      },
+      {
+        "name": "non-votee search+*right+*structure",
+        "type": "Search",
+        "content": "{\"type\":\"_2\",\"not\":{\"linked_to_by\":{\"left\":\"_user\",\"right\":[\"in\",\"*upvotes\",\"*downvotes\"]}},\"append\":\"_1\"}"
+      },
+      {
+        "name": "votee filter",
+        "type": "HTML",
+        "content": "<div class=\"votee-filter\">\r\n<span class=\"glyphicon glyphicon-filter\"></span>\r\nFilter\r\n<span class=\"filter-toggle\">\r\n<span class=\"glyphicon glyphicon-triangle-right\"></span>\r\n</span>\r\n<div class=\"filter-details\">\r\n<button class=\"btn btn-primary importance\">importance</button>\r\n<button class=\"btn btn-primary recent\">recent</button>\r\n</div>\r\n</div>"
+      },
+      {
+        "name": "Topic+Company+*ltype rtype+*structure",
+        "type": "Basic",
+        "content": ""
+      },
+      {
+        "name": "needs polish",
+        "type": "Pointer",
+        "content": "[[yinyang]]\n[[Company+metric sidebar+*type plus right+*structure]]\n[[Company+topic sidebar+*type plus right+*structure]]\n[[metric drag item]]\n[[topic drag item]]\n[[topic details+*right+*structure]]\n[[metric details+*right+*structure]]\n[[style: rating]]\n[[script: rating]]\n[[*upvotes+*empty list]]\n[[*downvotes+*empty list]]\n[[metric value timeline item]]"
+      },
```

Comment:
I collected all new cards that have something to do with the appearance in this pointer